Corona-Warn-App/src/main/res/values/contact_diary_strings.xml

```diff
@@ -60,7 +60,7 @@
     <!-- XTXT: Subtitle for contact diary overview screen -->
     <string name="contact_diary_overview_subtitle">"Write down the people you have met and places you have visited."</string>
     <!-- XTXT: Header for contact diary overview screen -->
-    <string name="contact_diary_overview_header">"Start Page"</string>
+    <string name="contact_diary_overview_header">Start Page</string>
```

Comment:
Why this change? Do we need to change this in other locales too?

Reply:
That just struck my eye in the preview, seems fine on device

Reply:
Note that any changes made to these strings will likely be overwritten by the next time translations are pulled.